### DISCUSSION TITLE
Job Filter Not Triggered

### DISCUSSION THREAD

**Discussion by itizarsa** [jobrunr/jobrunr#1013] [General]
I'm using the job filter like below based on the [docs](https://www.jobrunr.io/en/documentation/pro/job-filters), but the methods aren't getting triggered. Is there something I'm missing?

```
package com.example;

import java.time.Instant;

import org.jobrunr.jobs.Job;
import org.jobrunr.jobs.filters.JobServerFilter;
import org.springframework.stereotype.Component;

@Component
public class SchedulerFilter implements JobServerFilter {

	@Override
	public void onProcessing(Job job) {
		System.out.println(Instant.now());
	}

	public void onProcessingSucceeded(Job job) {
		System.out.println(Instant.now());
	}

	public void onProcessingFailed(Job job, Exception e) {
		System.out.println(Instant.now());
	}
}
```

**Comment by rdehuyss** [comment]
Job Filter beans are only supported in Pro.

**Reply by itizarsa** [reply]
This means we have to manually configure it using `Jobrunr.configure`?

Based on what I tried, it seems we can't have both `application.properties` & `Jobrunr.configure `together on the spring application. Is that right?
